python/tk_desktop/rpc.py

```diff
@@ -117,6 +216,7 @@ def run(self):
                     )
                     ready = True
```

Comment:
I know it's not one of your changes and I suspect the likely hood of this being an issue is low, but I think there is a chance that `ready` might not get defined, if it fails in the `try` with an error other than `WindowsError`.

Reply:
Good point. Fixed.